cloudevents/working-drafts/xml-format.md

```diff
@@ -135,7 +142,9 @@ Example:
 ### 3.3 XML Data
 
 XML data MUST be carried in an element with a defined type of `xs:any` with
-a single child XML element (with any mandatory namespace definitions).
+a exactly one child XML element (with any mandatory namespace definitions).
+The XML element MUST NOT contain any direct child text nodes with non-whitespace
```

Comment:
So, `<myevent>hello world</myevent>` is invalid?

Reply:
Whoops, I should clarify. (That's bad C&P.) It's meant to be that the `data` element must not contain any direct child text nodes.

Reply:
Updated in an additional commit.

Reply:
Could you post an example of invalid content ...what is the danger in the child xml element having such content ??